### ISSUE TITLE
EnC could skip next statement when active statement is commented out

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#3295]
Internally tracked as 1171881.

Repro: 
F5, comment out "Stop" statement, F10:

```VB
Module Module1                
Sub Main()                          
   Stop                                
   Console.WriteLine(1)       
   Console.WriteLine(2)          
End Sub
End Module
```

The debugger skips "Console.WriteLine(1)" and ends up stopping on "Console.WriteLine(2)".

**Comment by tmat** [comment]
Fixed by 044c1692fe14002184f4d43d99bccd374c3a5317

**Comment by tmat** [comment]
Verification is blocked by a debugger bug 1171881.

**Comment by tmat** [comment]
Fixed by 8acf54b34beb1da9b2ca297bef08b7f68372bda3
